components/ResourceInformation.js

```diff
@@ -0,0 +1,68 @@
+import { useOperation } from "react-openapi-client";
+import Button from "@mui/material/Button";
+import Card from "@mui/material/Card";
+import Chip from "@mui/material/Chip";
+import Rating from "@mui/material/Rating";
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import { Box, CircularProgress } from "@mui/material";
+import UserRating from "./UserRating";
+
+function ResourceInformation({ resourceId }) {
+  console.log(resourceId);
+  const { loading, data, error } = useOperation("getResource", resourceId);
```

Comment:
If the data obtained is a resource object, I think it would be cool to call it resource right away. You can do this replacing `data` for `data: resource`.
This way, you can later call `resource.average_evaluation`, `resource.name` and so on, and looks more natural

Reply:
Done
This file was moved to `component/resources` folder.